apis/sagemaker/v1beta2/zz_workteam_types.go

```diff
@@ -85,7 +85,7 @@ type CognitoMemberDefinitionParameters struct {
 
 	// An identifier for a user group.
 	// +crossplane:generate:reference:type=github.com/upbound/provider-aws/apis/cognitoidp/v1beta1.UserGroup
-	// +crossplane:generate:reference:extractor=github.com/crossplane/upjet/pkg/resource.ExtractResourceID()
+	// +crossplane:generate:reference:extractor=github.com/crossplane/upjet/pkg/resource.ExtractParamPath("name",false)
```

Comment:
Thanks for backporting the new fields, @mergenci; your tool seems to have done a good job 🙌 

I think we should just backport the comments line (code generation directives) changes in this file, I remember we talked about this. We can add these manually for now, but it would be good if the tool could handle these in the future 🙂

Reply:
Then let's attempt automating backporting of comments in the next Terraform provider bump. Thanks for the catch @turkenf. I backported documentation comments in addition to the directives you suggested. Please check 🙏